File: u03/src/Calculations.java

public class Calculations {
	public static boolean magic7(int a, int b) {
        return (a + b) == 7 || a > b && a - b == 7 || a < b && b - a == 7;
    }
	
	public static boolean fast12(int z) {
		int remainder = z % 12;
		return remainder <= 2 || remainder >= 10;
	}
}

File: u03/src/Calculations.cpp
#include <iostream>
using namespace std;

int magic7(int a, int b)
{
    return (a + b) == 7 || a > b && a - b == 7 || b > a && b - a == 7;
}

int fast12(int z)
{
    int r = z % 12;
    return r <= 2 || r >= 10;
}

int main()
{
    cout << magic7(56, 34) << endl;
    cout << fast12(12) << endl;

    return 0;
}